### ISSUE TITLE
How to install Opensesame 3.2.8 using Python3.x

### ISSUE THREAD

**Issue by ddetommaso** [s4hri/opensesame-docker#1]
I remember into your notes @adamlukomski you found a way to make it works.
Do you think it is possible to have everything in a Docker?

**Comment by adamlukomski** [comment]
Yes, I think I will have it ready tomorrow, I am currently at the themes download step. I'll update https://github.com/s4hri/organizational/blob/master/anaconda_psychopy_yarp_opensesame.md and https://github.com/s4hri/organizational/blob/master/opensesame.md too once I'm finished

**Comment by adamlukomski** [comment]
But again: psychopy will be as a selectable renderer-only thing, the GUI is still somewhat broken in that install, but that should not be an issue

**Comment by ddetommaso** [comment]
Ok. What I mean with "everything in one Docker" is that would be good to have a docker with only opensesame with python 3.x like this one https://github.com/s4hri/opensesame-docker 

and separately psychopy into another docker (but this is off-topic here)
